java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -316,6 +318,70 @@ public interface ServerManagementClusterCommands {
      */
     CompletableFuture<ClusterValue<Long>> lastsave(Route route);
 
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @see <a href="https://valkey.io/commands/flushall/">valkey.io</a> for details.
+     * @return <code>OK</code>.
+     * @example
+     *     <pre>{@code
+     * String response = client.flushall().get();
+     * assert response.equals("OK");
+     * }</pre>
+     */
+    CompletableFuture<String> flushall();
+
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @see <a href="https://valkey.io/commands/flushall/">redis.io</a> for details.
```

Comment:
Also needs to be changed in some of the other docs
```suggestion
     * @see <a href="https://valkey.io/commands/flushall/">valkey.io</a> for details.
```